### ISSUE TITLE
Fix file conflicts to resume image production

### ISSUE THREAD

**Issue by ermo** [serpent-os/recipes#8]
Turns out we had file conflicts and old moss just ignored them.

`util-linux` and `inetutils` are conflicting on a number of paths, including `usr/bin/logger`. Get these resolved.

Originally reported by @ikeycode

**Comment by der-eismann** [comment]
Hopefully #35 and #21 solved all the conflicts.

**Comment by der-eismann** [comment]
Ok, there's more. Built a quick PoC, hopefully I can set this up as a GitHub action:
```
INFO[0000] File: /usr/share/man/man8/fsck.8             
INFO[0000]   - Package: e2fsprogs                       
INFO[0000]   - Package: util-linux                      
INFO[0000] File: /usr/include/atk-1.0/atk/atknoopobjectfactory.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkhyperlink.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/share/gir-1.0/Atk-1.0.gir         
INFO[0000]   - Package: at-spi2-core                    
INFO[0000]   - Package: atk                             
INFO[0000] File: /usr/include/atk-1.0/atk/atkplug.h     
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/gtk-2.0/modules/libatk-bridge.so 
INFO[0000]   - Package: at-spi2-atk                     
INFO[0000]   - Package: at-spi2-core                    
INFO[0000] File: /usr/sbin/fsck                         
INFO[0000]   - Package: e2fsprogs                       
INFO[0000]   - Package: util-linux                      
INFO[0000] File: /usr/share/info/dir                    
INFO[0000]   - Package: bc                              
INFO[0000]   - Package: binutils                        
INFO[0000]   - Package: gcc                             
INFO[0000]   - Package: groff                           
INFO[0000] File: /usr/include/atk-1.0/atk/atkeditabletext.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkimage.h    
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkcomponent.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atktable.h    
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/gnome-settings-daemon-3.0/gtk-modules/at-spi2-atk.desktop 
INFO[0000]   - Package: at-spi2-atk                     
INFO[0000]   - Package: at-spi2-core                    
INFO[0000] File: /usr/include/atk-1.0/atk/atktext.h     
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkgobjectaccessible.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkobject.h   
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atksocket.h   
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atk.h         
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkmisc.h     
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkrelation.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkutil.h     
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkstateset.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkhyperlinkimpl.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/share/man/man8                    
INFO[0000]   - Package: dracut                          
INFO[0000]   - Package: inetutils                       
INFO[0000] File: /usr/include/atk-1.0/atk/atk-autocleanups.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/share/man/man5                    
INFO[0000]   - Package: dracut                          
INFO[0000]   - Package: file                            
INFO[0000] File: /usr/include/atk-1.0/atk/atkversion.h  
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkrelationtype.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkobjectfactory.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkvalue.h    
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkdocument.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkstreamablecontent.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/libatk-bridge-2.0.so          
INFO[0000]   - Package: at-spi2-atk-devel               
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000] File: /usr/share/info                        
INFO[0000]   - Package: e2fsprogs                       
INFO[0000]   - Package: screen                          
INFO[0000] File: /usr/include/atk-1.0/atk/atkaction.h   
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkregistry.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/libatk-bridge-2.0.so.0        
INFO[0000]   - Package: at-spi2-atk                     
INFO[0000]   - Package: at-spi2-core                    
INFO[0000] File: /usr/lib/libatk-1.0.so                 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkselection.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkstate.h    
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/libatk-1.0.so.0               
INFO[0000]   - Package: at-spi2-core                    
INFO[0000]   - Package: atk                             
INFO[0000] File: /usr/include/atk-1.0/atk/atkrange.h    
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/girepository-1.0/Atk-1.0.typelib 
INFO[0000]   - Package: at-spi2-core                    
INFO[0000]   - Package: atk                             
INFO[0000] File: /usr/include/atk-1.0/atk/atknoopobject.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/libatk-bridge-2.0.so.0.0.0    
INFO[0000]   - Package: at-spi2-atk                     
INFO[0000]   - Package: at-spi2-core                    
INFO[0000] File: /usr/include/atk-1.0/atk/atk-enum-types.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/bin/cpio                          
INFO[0000]   - Package: cpio                            
INFO[0000]   - Package: libarchive                      
INFO[0000] File: /usr/lib/pkgconfig/atk.pc              
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkhypertext.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/lib/pkgconfig/atk-bridge-2.0.pc   
INFO[0000]   - Package: at-spi2-atk-devel               
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000] File: /usr/include/atk-1.0/atk/atkrelationset.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/atk-1.0/atk/atkwindow.h   
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel                       
INFO[0000] File: /usr/include/at-spi2-atk/2.0/atk-bridge.h 
INFO[0000]   - Package: at-spi2-atk-devel               
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000] File: /usr/include/atk-1.0/atk/atktablecell.h 
INFO[0000]   - Package: at-spi2-core-devel              
INFO[0000]   - Package: atk-devel 
```

**Comment by der-eismann** [comment]
Some more fixes in #63
